common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.java

```diff
@@ -66,6 +69,16 @@
             BROKER_HOST_APP_SIGNATURE
     );
 
+    public static final BrokerData INTUNE_NGMS_AGENT_DEBUG = new BrokerData(
+            INTUNE_NGMS_AGENT_APP_PACKAGE_NAME,
```

Comment:
Can we drop the static external constants and just supply them locally here?  It doesn't make any sense at all to put these into AuthenticationConstants.

Reply:
but I think its better to have a single source of truth for such values.

Reply:
I could still make them local, but then keeping the values consistent across the codebase would be a problem,